en/frames_multicopter/dji_f450_cuav_5plus.md

```diff
@@ -0,0 +1,212 @@
+
+This topic provides full instructions for building the kit and configuring PX4 using *QGroundControl*.
+
+Key information
+
+- **Frame:** DJI F450
+- **Flight controller:** [CUAV V5+](../flight_controller/cuav_v5_plus.md)
+- **Assembly time (approx.):** 90 minutes (45 minutes for frame, 45 minutes autopilot installation/configuration)
+
+![Finished setup 1](../../assets/airframes/multicopter/dji_f450_cuav_5plus/f450_cuav5_plus_complete.jpg)
+
+## Bill of materials
+
+The components needed for this build are:
+* Power module (Holybro)
+* Fully assembled Power Management Board with ESCs
+* Motors - DR2205 KV2300
+* 5” Plastic Props
+* Carbon fiber 250 airframe with hardware
+* Foxer camera 
+* Vtx 5.8ghz 
+* Receiver FR SKY D4R-II
+* Battery 4S 1300 mAh
+* DJI F450 Frame
+
+Additionally you will need a battery and receiver (+compatible transmitter).
+This build uses:
+* Receiver: [FR SKY D4R-II](http://www.getfpv.com/radios/receivers/frsky-d4r-ii-4ch-2-4ghz-accst-receiver-w-telemetry.html)
+* Battery: [3S 5200mAh](TBD)
+
+The image below shows all the components used in this build.
+
+![All components used in this build](../../assets/airframes/multicopter/dji_f450_cuav_5plus/all_components.jpg)
+
+
+## Hardware
+
+Description | Quantity
+---|---
+Unibody Frame Plate           | 1
+Flight Controller Cover Plate | 1
+Camera Plate                  | 1
+35mm Standoffs                | 6
+Vinyl screws & Nuts           | 4
+15mm Steel Screws             | 8
+Steel Nuts                    | 8
+7mm Steel Screws              | 12
+Velcro Battery Strap          | 1
+Foam for Battery              | 1
+Landing Pads                  | 4
+
+## Package
+
+Description | Quantity (Default Package) | Quantity (GPS Package)
+--- | --- | ---
+V5+ Autopilot            | 1 | 1
+DuPont Cable             | 2 | 2
+I2C/CAN Cable            | 2 | 2
+ADC 6.6 Cable            | 2 | 2
+SBUS Signal Cable        | 1 | 1
+IRSSI Cable              | 1 | 1
+DSM Signal Cable         | 1 | 1
+ADC 3.3 Cable            | 1 | 1
+Debug Cable              | 1 | 1
+Safety Switch Cable      | 1 | 1
+Voltage & Current Cable  | 1 | 1
+PW-Link Module Cable     | 1 | 1
+Power Module             | 1 | 1
+SanDisk 16GB Memory Card | 1 | 1
+12C Expansion Board      | 1 | 1
+TTL Plate                | 1 | 1
+NEO GPS                  | - | 1
+GPS Bracket              | - | 1
+
+### Electronics
+
+Description | Quantity
+--- | --- 
+Motors - DR2205 KV2300             | 4
+Fully assembled Power Management Board with ESCs | 4
+Holybro power module               | 1
+Fr-sky D4R-II receiver             | 1
+CUAV V5+                           | 1
+Holybro GPS Neo-M8N                | 1
+Battery lumenier 1300 mAh 4S 14.8V | 1
+Vtx 5.8gHz                         | 1
+FPV camera                         | 1
+
+### Tools needed
+
+The following tools are used in this assembly:
+
+- 2.0mm Hex screwdriver
+- 3mm Phillips screwdriver
+- Wire cutters
+- Precision tweezers
+- Soldering iron
```

Comment:
@tony3dr FYI the images show soldering, so I added iron to the list and expanded the image to show this below. OK?